sklearn/model_selection/_search.py

```diff
@@ -649,8 +649,14 @@ def fit(self, X, y=None, groups=None, **fit_params):
 
         X, y, groups = indexable(X, y, groups)
         # make sure fit_params are sliceable
-        fit_params_values = indexable(*fit_params.values())
-        fit_params = dict(zip(fit_params.keys(), fit_params_values))
+        # TODO: remove in 0.24
+        try:
+            fit_params_values = indexable(*fit_params.values())
+            fit_params = dict(zip(fit_params.keys(), fit_params_values))
+        except TypeError:
+            warnings.warn("Support for scaler fit params is deprecated. "
```

Comment:
I'm not entirely convinced that we want to deprecate this support. If we do, we should also do for other cross validation routines.

Reply:
I don't think we should deprecate, cf https://github.com/scikit-learn/scikit-learn/issues/15805#issuecomment-565019797